pkgs/build-support/vm/windows/default.nix

```diff
@@ -22,7 +22,7 @@ in {
   runInWindowsVM = drv: let
     newDrv = drv.override {
       stdenv = drv.stdenv.override {
-        shell = "/bin/sh";
+        shell = "${stdenv.shell}";
```

Comment:
It definitely looks deliberate -- there is a further check `attrs.builder == attrs.stdenv.shell` when defining `origBuilder` later on. The file was originally written by @aszlig, if @aszlig could quickly shed some light here.